docs/modelview/modelview.rst

```diff
@@ -384,7 +430,6 @@ Having more delegates sacrifices memory for a smoother experience and slightly m
 
 To remedy the two later issues, it is recommended to use ``Loader`` elements. These can be used to instantiate additional elements when they are needed. For instance, an expanding delegate may use a ``Loader`` to postpone the instantiation of its detailed view until it is needed. For the same reason, it is good to keep the amount of JavaScript to a minimum in each delegate. It is better to let them call complex pieced of JavaScript that resides outside each delegate. This reduces the time spent compiling JavaScript each time a delegate is created.
```

Comment:
later or latter?

Reply:
latter

Putting too many Loaders into delegates is one thing that made the Controls 1 TableView perform badly.  I would do it only when a delegate is so big that it slows down flicking, or the delegate depends on remotely-fetched data, has media to render, or something like that; but not as a matter of course.  (this comment is just a comment, not for direct pasting ;-)

Reply:
Good points. I've updated accordingly.